lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/BitmapHelperFunctions.java

```diff
@@ -18,7 +18,7 @@
      */
     public static void updateAggregates(FastFixedHashTable thisThreadSetAggregates,
                                         IntSet curCandidate, AggregationOp[] aggregationOps,
-                                  double[] aggregateVal, int numAggregates) {
```

Comment:
Can you discard all the changes in this file, since it's strictly whitespace changes?

Reply:
Are you sure you're comparing the most recent versions?  There should be big content changes.

Reply:
Oh, yeah—it was out of date